cli.py

```diff
@@ -12,6 +12,7 @@
 from ingress.db_updater import DbUpdater
 from ingress.embeddings import EmbeddingTrainer
 from ingress.helpers import JURISDICTIONS
+from ingress.scrape_citation_contexts import CitationContextScraper
```

Comment:
I think we should name the file citation_context_scraper for consistency?

Reply:
Good call